CHANGES.rst

```diff
@@ -8,11 +8,21 @@
 Changes
 =======
 
+Version 1.4.1 (released TBD)
+
+- set Sphinx ``<3`` because of errors related to application context
```

Comment:
minor: what about 3.0.0, just for this case and the heart emoji...? I know quoted it doesn't translate but still. Don't want to trigger travis tho.

Reply:
Again as per the documentation training, I was seeking for simplicity as I don't see the "emoji" as a problem, rather a slightly funny coincidence.